docs/pages/database-access/guides/aws-opensearch.mdx

```diff
@@ -64,7 +56,7 @@ does not require any permissions. In this guide, we will use the example name
 `TeleportDatabaseService` for this role. Once you have chosen a name, click
 **Create Role** to complete the process.
 
-### OpenSearch Mange Cluster access role
+### OpenSearch Manage Cluster access role
```

Comment:
```suggestion
### OpenSearch Service cluster access role
```
Looks like "OpenSearch Service" is the product name, not "OpenSearch Manage" (https://aws.amazon.com/opensearch-service/).